docs/src/main/docs/guides/91_mp-tutorial.adoc

```diff
@@ -0,0 +1,1258 @@
+ https://microprofile.io/project/eclipse/microprofile-metrics[MicroProfile Metrics
+ Specification].
+
+[source,java]
+.Updated GreetResource.java with custom metrics
+----
+import org.eclipse.microprofile.metrics.annotation.Timed;
+
+    // some lines omitted
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @Timed <1>
+    public JsonObject getDefaultMessage() {
+        return createResponse("World");
+    }
+----
+
+<1> The `Timed` annotation is added to the `getDefaultMessage()` method.
+
+Rebuild and run the application.  Make some calls to the endpoint 
+ (http://localhost:8080/greet) so there will be some data to report.
+ Then obtain the application metrics as follows:
+
+[source,bash]
+.Checking the application metrics
+----
+curl -H "Accept: application/json" http://localhost:8080/metrics/application
+{
+  "io.helidon.examples.GreetResource.getDefaultMessage": {
+    "count": 2,
+    "meanRate": 0.036565171873527716,
+    "oneMinRate": 0.015991117074135343,
+    "fiveMinRate": 0.0033057092356765017,
+    "fifteenMinRate": 0.0011080303990206543,
+    "min": 78658,
+    "max": 1614077,
+    "mean": 811843.8728029992,
+    "stddev": 766932.8494434259,
+    "p50": 78658,
+    "p75": 1614077,
+    "p95": 1614077,
+    "p98": 1614077,
+    "p99": 1614077,
+    "p999": 1614077
+  }
+}
+----
+
+Learn more about using Helidon and MicroProfile metrics in the <<guides/09_metrics_mp_guide.adoc,Metrics Guide>>.
+
+
+== Health Check
+
+Helidon provides built-in support for health check endpoints.  Obtain 
+ the built-in health check using the following URL: 
+
+[source,bash]
+.Health check
+----
+curl -s -X GET http://localhost:8080/health
+{
+  "outcome": "UP",
+  "status": "UP",
+  "checks": [
+    {
+      "name": "deadlock",
+      "state": "UP",
+      "status": "UP"
+    },
+    {
+      "name": "diskSpace",
+      "state": "UP",
+      "status": "UP",
+      "data": {
+        "free": "381.23 GB",
+        "freeBytes": 409340088320,
+        "percentFree": "43.39%",
+        "total": "878.70 GB",
+        "totalBytes": 943491723264
+      }
+    },
+    {
+      "name": "heapMemory",
+      "state": "UP",
+      "status": "UP",
+      "data": {
+        "free": "324.90 MB",
+        "freeBytes": 340682920,
+        "max": "3.46 GB",
+        "maxBytes": 3715629056,
+        "percentFree": "97.65%",
+        "total": "408.00 MB",
+        "totalBytes": 427819008
+      }
+    }
+  ]
+}
+----
+
+Endpoints for readiness and liveness checks are also provided by default.
+ Obtain the default results using these URLs:
+
+[source,bash]
+.Default readiness and liveness endpoints
+----
+# readiness
+curl -i  -X GET http://localhost:8080/health/ready
+HTTP/1.1 200 OK <1>
+Content-Type: application/json
+Date: Thu, 22 Aug 2019 13:06:37 -0400
+connection: keep-alive
+content-length: 42
+
+{"outcome":"UP","status":"UP","checks":[]} <2>
+
+# liveness
+curl -i  -X GET http://localhost:8080/health/live
```

Comment:
This is no longer true - in the current version, all default health checks are also liveness health checks.

Reply:
not sure what i need to change

Reply:
If you try the stuff you are doing on current master (and maybe on 1.2.1), you would not get an empty result for liveness checks - you would get the same result as for `/health`